server/src/types/course.ts

```diff
@@ -45,6 +45,13 @@ export interface CourseInstanceData {
   endDate: Date,
   teachingMethod: TeachingMethod,
   gradingType: GradingType,
-  responsibleTeacher?: string | undefined,
-  responsibleTeachers?: Array<string>,
+  // TODO: There should maybe be a UserData interface if more data is needed,
+  // for example ID of user.
+  teachersInCharge?: Array<string>
+}
+
+export enum CourseInstanceRoleType {
+  Student = 'STUDENT',
+  Teacher = 'TEACHER',
+  TeacherInCharge = 'TEACHERINCHARGE',
```

Comment:
I think this is just a matter of taste , but should we use underscore as word separator in `TEACHERINCHARGE` to improve readability? (Also in the database and sequelize models)

Reply:
Yes, that would be better. The reason I chose `TEACHERINCHARGE` was because it seemed more consistent with `PASSFAIL`.

Reply:
Would it then make sense to also update the `PASSFAIL` to `PASS_FAIL`

Reply:
Changed both. I also decided update some other naming we've decided to change before.